.github/workflows/main.yml

```diff
@@ -37,22 +37,23 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install dependencies
-        run: >-
-          sudo apt install -y
-          build-essential
-          php7.4-cli
-          haskell-platform
-          fp-compiler
-          python2.7
-          cppreference-doc-en-html
-          cgroup-lite
-          libcap-dev
-          python-dev
-          libpq-dev
-          libcups2-dev
-          libyaml-dev
-          rustc
-          mono-mcs
+        run: |
+          sudo apt update -y
```

Comment:
While we are changing this, could you please also replace `apt` with `apt-get`? `apt` gives
```
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
```

Reply:
Ups, you're right, I'll fix this in the next commit.